src/Apps/Artwork/Components/__tests__/ArtworkDetails.test.tsx

```diff
@@ -17,6 +18,7 @@ describe("ArtworkDetails", () => {
           }
         }
       `,
+      wrapper: n => <MockBoot breakpoint="xs">{n}</MockBoot>,
```

Comment:
We can do this in another PR but should pop off the `breakpoint='xs'` and defer to defaults